components/renku_data_services/session/api.spec.yaml

```diff
@@ -531,11 +641,105 @@ components:
       description: A description for the resource
       type: string
       maxLength: 500
+    ImageBuilder:
+      description: A container image builder
+      type: object
+      additionalProperties: false
+      properties:
+        builder_id:
+          $ref: "#/components/schemas/Ulid"
+        container_image:
+          $ref: "#/components/schemas/ContainerImage"
+        repository:
+          $ref: "#/components/schemas/Repository"
+        branch:
+          $ref: "#/components/schemas/branch"
```

Comment:
This can probably be any github ref, not just branches? In shipwright builds this is called `revision`